.vscode/launch.json

```diff
@@ -24,6 +24,18 @@
                 "gazetteer.geojson",
                 "location_aliases.csv"
             ]
+        },
+        {
+            "name": "_safedata_validator_cli debug",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "args": [
+                "-r",
+                "local/config_files/live_safeweb_gbif_2016_07_25.cfg",
```

Comment:
Should specific paths be added here?

Reply:
I've unpacked the explanation a bit more to make the transition from what you'd run in the shell to how it is setup to debug via python in `vscode`.